catho/quantum
# build(QTM-729): Fix for styling issues for components that use the same css file name

## Description
https://jirasoftware.catho.com.br/browse/QTM-729

## How to test

Remove the "Modal" prefix from the css styling files of Modal subcomponents, in the components/Modal/sub-components folder. Ex: ModalContent.modules.css should be Content.module

Run 'yarn test:regression:storybook' and see if the component's styles are preserved

Check that the Card.Content component styles are preserved

Run the build, install it in srr-simulator and check if the styles are as implemented by inspecting the page in the browser

## Review guide
